### ISSUE TITLE
How to add class to open parent menu, when submenu is open

### ISSUE THREAD

**Issue by qubiack** [isaacplmann/ngx-contextmenu#157]
When I go into submenu, parent menu doesn't have any calss to style them, like in system right click menu to keep logic path, for example:
when I go into "Say hi! -> Edit" I want change background under "Say hi!" to show path for user.

https://stackblitz.com/edit/ngx-contextmenu-example-dz26wq

**Comment by isaacplmann** [comment]
You’re right.  It would be a nice feature to add an `active-path` class to an item that has an open sub menu.

I’d accept a PR for that feature.
